journal/original/10-chapter-8.adoc

```diff
@@ -126,10 +131,16 @@ and bring them to the blessed sheepfold of Christ Jesus, is the desire of him,
 who is a travailer in body and spirit for the outcasts of Israel, and scattered of Jacob,
 that they may be gathered.
 
+[.signed-section-signature]
 Charles Marshall.
 
+[.embedded-content-document.epistle]
+--
+
+[.letter-heading]
```

Comment:
On the modernized, you marked this as a blurb. I think they both look good, but they should probably be the same in both versions. So maybe mark this one a blurb too, instead of a letter-heading. 👍🏻